lib/models/upgrade_info.dart

```diff
@@ -37,3 +37,21 @@ enum ProrationMode {
   /// plus remaining prorated time from the old plan.
   immediateAndChargeFullPrice
 }
+
+// ignore_for_file: deprecated_member_use_from_same_package
+extension ProrationModeExtension on ProrationMode {
```

Comment:
@tonidero Are these values correct? Just want to double check 😅

Reply:
These values look good! But I just noticed we are missing `deferred` as well, which should have a value of `6`. Thanks for double-checking! I see I forgot to add that in my base PR...

Reply:
Oh wait! There is also the `GoogleProductChange` that has `deferred` with the correct value. This one is deprecated, so no need to add `deferred` here. Nvm then! These look good :)